include/nuttx/virtio/virtio-net.h

```diff
@@ -0,0 +1,68 @@
+/****************************************************************************
+ * include/nuttx/virtio/virtio-net.h
```

Comment:
could move to drivers/virtio since it is only called by virtio-mmio.c

Reply:
done